### PR TITLE
Update Identity FAQ with correct Android setup instructions

### PR DESCRIPTION
**Pull request by kevinlind**
## Description

Update Identity for Edge Network FAQ page by correcting Android installation instructions when including both Identity extensions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/edge/identity-for-edge-network/tabs/faq.md [hunk 1]
```diff
@@ -97,7 +113,11 @@ class MobileApp : Application() {
         MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID)
         // Register Adobe Experience Platform SDK extensions
         MobileCore.registerExtensions(
-            listOf(Edge.EXTENSION, Identity.EXTENSION)
+            listOf(
+                Edge.EXTENSION, 
+                com.adobe.marketing.mobile.edge.identity.Identity.EXTENSION,
+                com.adobe.marketing.mobile.Identity.EXTENSION
```
Comment (cacheung) [inline src/pages/edge/identity-for-edge-network/tabs/faq.md:119]: can we list the direct identity first, then edge identity same as the Java example?

Comment (kevinlind) [reply src/pages/edge/identity-for-edge-network/tabs/faq.md:119]: I did the reverse. I updated this Kotlin example to have the Edge Identity extension listed first.

File: src/pages/edge/identity-for-edge-network/tabs/faq.md
Comment (addb) [inline src/pages/edge/identity-for-edge-network/tabs/faq.md:1]: Should we link this FAQ to our main doc either on core/identity or edgeidentity or both overview page registerExtension sections 
ex: https://developer.adobe.com/client-sdks/edge/identity-for-edge-network/#add-the-identity-extension-to-your-app

Comment (cacheung) [reply src/pages/edge/identity-for-edge-network/tabs/faq.md:1]: We can do that but in a way which it won't confuse the users thought they need to install both identity extensions.

**Review by cacheung — APPROVED** [review]

File: src/pages/edge/identity-for-edge-network/tabs/faq.md
Comment (kevinlind) [reply src/pages/edge/identity-for-edge-network/tabs/faq.md:1]: There is already a note and link to the FAQ in the Edge Identity page you linked. 

> The following instructions are for configuring an application using Adobe Experience Platform Edge mobile extensions. If an application will include both Edge Network and Adobe Solution extensions, both the Identity for Edge Network and Identity for Experience Cloud ID Service extensions are required. Find more details in the [frequently asked questions](https://developer.adobe.com/client-sdks/edge/identity-for-edge-network/faq/).

**Review by addb — APPROVED** [review]

**Pull request merged**